.github/workflows/test.yml

```diff
@@ -80,11 +84,11 @@ jobs:
 
     - name: >-
         Initialize tox envs
-      run: >-
-        python3 -m tox --notest --skip-missing-interpreters false -vv
-      env: ${{ matrix.env }}
+      run: |
+        set -ex
+        ${{ matrix.PREFIX }} tox -e ${{ matrix.tox_env }} --notest --skip-missing-interpreters false -vv
```

Comment:
Drop the explicit env as suggested above (by the way, it'll cause a potential bug if you add a space after the comma in the matrix factors, this is why you should always watch out for quoting when injecting such things like this) + drop non-existing `PREFIX`.

Reply:
done as suggested